front/src/applications/stdcm/components/StdcmForm/StdcmConfig.tsx

```diff
@@ -180,6 +207,50 @@ const StdcmConfig = ({
     };
   }, [showMessage, formErrors]);
 
+  useEffect(() => {
```

Comment:
We can simplify the useEffect like this (it seems to work fine)

```suggestion
  useEffect(() => {
    if (
      formErrors?.errorType !== StdcmConfigErrorTypes.MISSING_INFORMATIONS ||
      !formErrors.errorDetails?.missingFields
    ) {
      return;
    }

    const updatedMissingFields = filterMissingFields({
      missingFields: formErrors.errorDetails.missingFields,
      rollingStock,
      totalMass,
      totalLength,
      maxSpeed,
      origin,
      destination,
    });

    if (updatedMissingFields.length === 0 && formErrors) {
      return;
    }

    if (updatedMissingFields.length === 0 && !formErrors) {
      setFormErrors(undefined);
      return;
    }

    setFormErrors({
      errorType: StdcmConfigErrorTypes.MISSING_INFORMATIONS,
      errorDetails: {
        missingFields: updatedMissingFields,
      },
    });
  }, [rollingStockId, totalMass, totalLength, maxSpeed, origin, destination]);
```

To only thing we have to think about is what behavior we want when we have both error types at the same time (missing field + other). I asked @thibautsailly about that, waiting for his answer

Reply:
Warnings should be displayed as different sections of the same message bloc, in this priority order:
1. Origin and destination timing preferences incompatibility
2. Information invalidity problems
3. Missing informations

Each section should order their warning in the reading order of the form: consist, origin, intermediate OPs, destination.

Reply:
This will be handle in another PR. Could you create the issue @Caracol3 ?

Reply:
Yes, it will be handled in https://github.com/OpenRailAssociation/osrd/issues/11278